staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go

```diff
@@ -166,8 +165,6 @@ func (s *WatchServer) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	metrics.RegisteredWatchers.WithContext(req.Context()).WithLabelValues(kind.Group, kind.Version, kind.Kind).Inc()
 	defer metrics.RegisteredWatchers.WithContext(req.Context()).WithLabelValues(kind.Group, kind.Version, kind.Kind).Dec()
 
-	w = httplog.Unlogged(req, w)
```

Comment:
why is this gone?

Reply:
The removal of this line fixes the bug where `code` is always `0` for a watch request in audit log, httplog, and metric. 
``` 
apiserver_request_total{code="0",component="apiserver",dry_run="",group="",resource="configmaps",
scope="namespace",subresource="",verb="WATCH",version="v1"} 37
```

Because watch.go uses the original `ResponseWriter` object the decorators provided by audit/httplog/metric do not get invoked, I explained this in the PR description in details